gnat2goto/driver/tree_walk.adb

```diff
@@ -1723,9 +1709,10 @@ package body Tree_Walk is
          if Nkind (E) in N_Has_Etype and then (Is_Type (Etype (E))) then
             return Do_Defining_Identifier (E);
          end if;
-         Report_Unhandled_Node_Empty (N, "Do_Identifier",
-                                         "Etype not a type");
-         return R : constant Irep := New_Irep (I_Symbol_Expr);
```

Comment:
This was previously returning a `Symbol_Expr`. Why did this change?

Reply:
There isn't really a way to create a valid symbol expr without setting its name and type, but we can't do this here because this is an error case.